docker/launch_backend_service.sh

```
@@ -148,7 +148,7 @@ run_admin_server(){
     if [[ "${API_PROXY_SCHEME}" == "go" ]]; then
         prepare_for_go
         server_name="admin_server"
-        server_cmd=("bin/admin_server")
+        server_cmd=("bin/ragflow_server", "--admin")
```

Shellcheck (0.11.0)

[warning] Use spaces, not commas, to separate array elements.

(SC2054)